crud/select/compat/select.lua

```diff
@@ -173,13 +175,21 @@ local function build_select_iterator(vshard_router, space_name, user_conditions,
         yield_every = yield_every,
         fetch_latest_metadata = opts.call_opts.fetch_latest_metadata,
     }
+    local merger
 
-    local merger = Merger.new(vshard_router, replicasets_to_select, space, plan.index_id,
-            common.SELECT_FUNC_NAME,
-            {space_name, plan.index_id, plan.conditions, select_opts},
-            {tarantool_iter = plan.tarantool_iter, field_names = plan.field_names, call_opts = opts.call_opts}
-        )
-
+    if opts.readview then
+        merger = Merger.new_readview(vshard_router, replicasets_to_select, opts.readview_uuid,
+        space, plan.index_id, '_crud.select_readview_on_storage',
```

Comment:
Put `'_crud.select_readview_on_storage'` to a variable, similar to `common.SELECT_FUNC_NAME`